CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Adds GitLab projects integration.
+- Adds CERN SSO integration and login/logout handling.
+- Adds state management with Redux.
+- Improves UX in projects page.
+- Allows enabling/disabling GitLab project integration easily.
```

Comment:
What do you think about ``Allows enabling/disabling GitLab project integration.``? Is there a need to put emphasis on `easily`?